core/chains/evm/chain_set.go

```diff
@@ -425,69 +260,12 @@ type ChainSetOpts struct {
 	GenTxManager      func(*big.Int) txmgr.TxManager
 }
 
-// https://app.shortcut.com/chainlinklabs/story/33622/remove-legacy-config
-func LoadChainSet(ctx context.Context, opts ChainSetOpts) (ChainSet, error) {
-	if err := opts.check(); err != nil {
-		return nil, err
-	}
-	if h, ok := opts.Config.(v2.HasEVMConfigs); ok {
-		return NewTOMLChainSet(ctx, opts, h.EVMConfigs())
-	}
-
-	chains, err := opts.ORM.EnabledChains()
-	if err != nil {
-		return nil, errors.Wrap(err, "error loading chains")
-	}
-	nodesSlice, _, err := opts.ORM.Nodes(0, -1)
-	if err != nil {
-		return nil, errors.Wrap(err, "error loading nodes")
-	}
-	nodes := make(map[string][]types.Node)
-	for _, n := range nodesSlice {
-		id := n.EVMChainID.String()
-		nodes[id] = append(nodes[id], n)
-	}
-	return NewDBChainSet(ctx, opts, chains, nodes)
-}
-
-// NewDBChainSet returns a new ChainSet from legacy configuration.
-// https://app.shortcut.com/chainlinklabs/story/33622/remove-legacy-config
-func NewDBChainSet(ctx context.Context, opts ChainSetOpts, dbchains []types.DBChain, nodes map[string][]types.Node) (ChainSet, error) {
-	if err := opts.check(); err != nil {
-		return nil, err
-	}
-	opts.Logger = opts.Logger.Named("EVM")
-	defaultChainID := opts.Config.DefaultChainID()
-	if defaultChainID == nil && len(dbchains) >= 1 {
-		defaultChainID = dbchains[0].ID.ToInt()
-		if len(dbchains) > 1 {
-			opts.Logger.Debugf("Multiple chains present but ETH_CHAIN_ID was not specified, falling back to default chain: %s", defaultChainID.String())
-		}
-	}
-	var err error
-	cll := newChainSet(opts)
-	cll.defaultID = defaultChainID
-	for i := range dbchains {
-		cid := dbchains[i].ID.String()
-		cll.logger.Infow(fmt.Sprintf("Loading chain %s", cid), "evmChainID", cid)
-		chain, err2 := newDBChain(ctx, dbchains[i], nodes[cid], opts)
-		if err2 != nil {
-			err = multierr.Combine(err, err2)
-			continue
-		}
-		if _, exists := cll.chains[cid]; exists {
-			return nil, errors.Errorf("duplicate chain with ID %s", cid)
-		}
-		cll.chains[cid] = chain
-	}
-	return cll, err
-}
-
 // NewTOMLChainSet returns a new ChainSet from TOML configuration.
-func NewTOMLChainSet(ctx context.Context, opts ChainSetOpts, chains []*v2.EVMConfig) (ChainSet, error) {
+func NewTOMLChainSet(ctx context.Context, opts ChainSetOpts) (ChainSet, error) {
```

Comment:
nit: does it make sense to rename this NewChainSet since TOML is redundant now we don't have another way?

Reply:
Maybe, I'll look again in the follow up.